packages/harpparts/src/access-parts/constants/tunings.ts

```diff
@@ -1,4 +1,4 @@
-import { TuningIds } from '../../tuning'
+import { SIXTEEN_HOLE_CHROMATIC, TuningIds } from '../../tuning'
```

Comment:
Don't forget to add some more chromatics too, including the diminished one which was requested.

Reply:
I think that was specfically a seydel nonslide one. Check the email on it.

Reply:
Added as techdebt: https://github.com/js-jslog/harpguru/issues/131#issuecomment-985646857